contracts/implementations/erc20Pay/RMRKErc20Pay.sol

```diff
@@ -7,13 +7,26 @@ import "./IRMRKErc20Pay.sol";
 
 error RMRKNotEnoughAllowance();
 
+/**
+ * @title RMRKNestable
+ * @author RMRK team
+ * @notice Smart contract of the RMRK Nestable module.
+ * @dev This contract is hierarchy agnostic and can support an arbitrary number of nested levels up and down, as long as
```

Comment:
Do you think this is an important clarification? It doesn't feel a critical piece (I never really thought of this concept here)

Reply:
We can just scrap it :D